packages/gui/src/constants/plotSizes.ts

```diff
@@ -43,9 +55,23 @@ export const plottingInfo: Record<PlotterName, PlotSize[]> = {
   ],
 };
 
-export function getPlotSizeOptions(plotterName: PlotterName) {
-  return plottingInfo[plotterName].map((item) => ({
-    value: item.value,
-    label: `k=${item.value} (Effective plot size: ${item.effectivePlotSize}, Temporary space: ${item.workspace})`,
-  }));
+export function getPlotSizeOptions(plotterName: PlotterName, compressionLevel?: number) {
+  return plottingInfo[plotterName].map((item) => {
+    const kSize = item.value;
+    if (
+      typeof compressionLevel !== 'number' ||
+      !compressedSizes[compressionLevel] ||
+      !compressedSizes[compressionLevel][kSize]
+    ) {
+      return {
+        value: kSize,
+        label: `k=${kSize} (Effective plot size: ${item.effectivePlotSize}, Temporary space: ${item.workspace})`,
```

Comment:
Is this label being localized somewhere else? Same question for the label on line 74